teku/src/test/java/tech/pegasys/teku/cli/BeaconNodeCommandTest.java

```diff
@@ -203,7 +203,9 @@ private Path createConfigFile() throws IOException {
     final URL configFile = this.getClass().getResource("/complete_config.yaml");
     final String updatedConfig =
         Resources.toString(configFile, UTF_8)
-            .replace("data-path: \".\"", "data-path: \"" + dataPath.toString() + "\"");
+            .replace(
+                "data-path: \".\"",
+                "data-path: \"" + dataPath.toString().replace("\\", "\\\\") + "\"");
```

Comment:
What's this for?

Reply:
This the test fix for Windows

Reply:
The `\` in win paths should be escaped in the YAML